cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -226,16 +223,20 @@ def chargeback_costs_default(resource_usage, db, provider):
     average_disk_io = resource_usage['average_disk_io']
 
     cpu_rate = query_rate(db, provider, 'cpu_usagemhz_rate_average', 'Default', 'Compute')
-    cpu_used_cost = average_cpu_used_in_mhz * float(cpu_rate) / 24
+    cpu_used_cost = version.pick({version.LOWEST: average_cpu_used_in_mhz * float(cpu_rate),
```

Comment:
It seems the functionality is common here, instead of vpicks how about this
```python
def convert_rate(rate):
    if version.current_version() > 5.7:
        return float(rate) * 24
    else:
        return float(rate)
```

Then usage becomes

```
    memory_used_cost = average_memory_used_in_mb * convert_rate(memory_rate)
```